dev/io.openliberty.restfulWS.mpMetrics.filter/src/io/openliberty/restfulws/mpmetrics/MonitorAppStateListener.java

```diff
@@ -109,37 +75,4 @@ public void setMonitorMetricsHandler(MonitorMetricsHandler monitorMetricsHandler
         monitorMetricsHandler = monitorMetricsHandlerService;
     }
 
-    @Activate
-    protected void activate(ComponentContext context, Map<String, Object> properties) {
```

Comment:
This is related to disabling the filter if the "REST" string is not part of the `filter` attribute for the `monitor` element in serer.xml. But since we are removing the filter and using a callback mechanism instead, its no longer needed here.